README.md

```diff
@@ -5,22 +5,51 @@ The creators of the below listed watchfaces are happy to answer your questions a
 Feel free to pull request your work here and if it suits the graphic guidelines of AsteroidOS, it can eventually be merged into the default set of watchfaces shipped with asteroid-launcher.
 
 
-### Install selected or all watchfaces using scripted SCP/ADB ###
+### Install selected or all watchfaces
 
 - Open a terminal and clone this unofficial-watchfaces repo to a new subfolder from your current location.\
 `git clone https://github.com/AsteroidOS/unofficial-watchfaces`
 - Cd into unofficial-watchfaces folder.\
 `cd unofficial-watchfaces/`
 - Connect your AsteroidOS Watch, configured to either ADB Mode (ADB transfer) or Developer Mode (SCP transfer) in Settings -> USB.
-- Start `./deploy.sh` to use SCP commands or `./deploy.sh -a` for ADB commands.
-- You can also use `./deploy.sh --help` to get a list of available options.
-- Select a single watchface to deploy to the watch with its given number or copy all available watchfaces at once with option 1.
-- You can then restart the ceres session and apply a single selected watchface with pressing 'y'.
+- Start `./watchface` to use SCP commands or `./watchface -a` for ADB commands.
+- You can also use `./watchface --help` to get a list of available options.
+- Select a single watchface to deploy to the watch with its given number or copy multiple watchfaces at once
+- The first selected watchface (if more than one is selected) will be activated
 
 Note that restarting the ceres session might be necessary when new fonts were installed along with the new watchfaces.
 Restarting the ceres session might break things like Always On Mode or the battery display for the remaining uptime. Reboot the watch in that case.
-You may [restart the session or reboot the watch](https://asteroidos.org/wiki/useful-commands/#restart) manually.
+You may [restart the session or reboot the watch](https://asteroidos.org/wiki/useful-commands/#restart) manually, or you may use the `--boot` option of `watchface`.
```

Comment:
Restarting the session is done automatically ?

Reply:
Yes, when the user selects "deploy" restarting the session is done automatically.  I've changed the README to be a little more clear about that.